rts/lra/lra-coordinator-jar/src/main/java/io/narayana/lra/coordinator/domain/service/LRAService.java

```diff
@@ -189,13 +188,9 @@ public void remove(String state, URI lraId) {
             lras.remove(lraId);
         }
 
-        if (recoveringLRAs.containsKey(lraId)) {
-            recoveringLRAs.remove(lraId);
-        }
+        recoveringLRAs.remove(lraId);
```

Comment:
was this part a general tidy up or was it related to JBTM-3243? if general I think it needs a separate issue?

Reply:
Really! Perhaps if I remember I will include it in a general tidy up sometime.

Reply:
It's not caused by https://issues.redhat.com/browse/JBTM-3243 is it? But as our LRA implementation is not formally released perhaps just a separate commit would suffice?

Reply:
The problem with having to raise a JBTM against every trivial issue on a code base that is tracking an evolving pre-release spec is that is slows down progress to delivering a first cut of the s/w

Reply:
yeah, I tend to agree for the tidy up stuff. a separate commit though would be reasonable as it's not right to include in a commit with a JBTM that does not apply, WDYT?

Reply:
Sure, a separate commit in the same PR would work for me (not in this one though since I reverted the change).